networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/gossip/config/GossipConfig.java

```diff
@@ -32,9 +32,9 @@ public class GossipConfig {
   private static final int DEFAULT_ADVERTISE = 3;
   private static final int DEFAULT_HISTORY = 6;
   static final Duration DEFAULT_HEARTBEAT_INTERVAL = Duration.ofMillis(700);
-  // ATTESTATION_PROPAGATION_SLOT_RANGE needs to be the last 32 slots, so TTL is 33 slots
-  // 566 * HEARTBEAT == 396.2 seconds, 33 slots is (396 seconds) on mainnet
-  static final Duration DEFAULT_SEEN_TTL = DEFAULT_HEARTBEAT_INTERVAL.multipliedBy(566);
+  // In Deneb, attestations are valid for up to 2 full epochs, so TTL is 65 slots
```

Comment:
I would just clarify the comment "After EIP-7045, attestations are valid for up to 2 full epochs..."